extra_data/write_cxi.py

```diff
@@ -179,60 +176,29 @@ def _map_chunk(self, chunk, chunk_data, target, tgt_ax1, have_data=None):
             chunk_match_start = chunk_match_end
             chunk_tids = chunk_tids[n_match:]
 
-    def collect_data(self):
+    def _map_layouts(self, layouts):
         """
-        Prepare virtual layouts and map them to the virtual sources in
-        the data chunks.
+        Map virtual layouts to the virtual sources in the data chunks.
```

Comment:
In my mind, this goes the other way - you map sources into the layouts. :thinking:

Reply:
Thank you, my English is far from perfect, so I appreciate corrections) Modified in https://github.com/European-XFEL/EXtra-data/commit/6582e9954511eb0bc752851d8f64a31bb7ed1348.